src/Blockcore.Indexer.Core/Models/QueryAddressBalance.cs

```diff
@@ -5,4 +5,8 @@ public class QueryAddressBalance
    public string Address { get; set; }
 
    public long Balance { get; set; }
+
+   public long PendingSent { get; set; }
```

Comment:
The name is a bit confusing, perhaps pending inbound and pending outbount?

Reply:
This is the name we use in the other endpoint, I think its better then inbound becuase it is clear it is a payment